### PR TITLE
Changelog: update for release

### PR DESCRIPTION
**Pull request by jeherve**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -106,52 +106,53 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
-* WordPress.com REST API: add a new API endpoint that will allow us to enable translation autoupdates. #5953
-* WordPress.com REST API: add a new API endpoint for pulling post rows and metadata for backups. #6230
-* WP REST API: retrieve all module settings in the readable `/settings` endpoint. #6002
+* Keep an eye our for a note from WooCommerce on how your e-commerce store may benefit from our new USPS and CanadaPost shipping functionality.
+* We've added an error message if Publicize isn't functioning as it should be.
+* The Twitter Widget timeline now displays the tweet limit count as 20, prior to this it appeared to have no limit.
+
+**Slightly Less Exciting Enhancements:**
+
+* In an effort to help us better understand features that are being used Twitter's timeline widget and Twitter's shortcodes now pass Jetpack's partner ID.
```
Comment (dereksmart) [inline readme.txt]: Feel like there should be a comma here `being used, Twitter's timeline...`

File: readme.txt [hunk 2]
```diff
@@ -106,52 +106,53 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
-* SSO: Ensure that logging in via SSO doesn't stop you authenticating to your site using a third-party application relying on the WordPress.com REST API. #6194
-* Sync: avoid PHP notices with the Shortcodes module. #6115, #6151, #6153
-* Tonesque: avoid PHP warnings when an image doesn't exist. #6068
-* Twitter Widget: add noscrollbar option back. #6131
-* Top Posts Widget: fix image size when using the List Layout. #6140
-* WordPress.com REST API: fix notices and warnings when updating data from the WordPress.com central interface. #6104
+* Some sites were using illegal multibyte characters and failing to sync posts, this has been fixed.
+* IE11 was giving our admin layout a bit of a problem but we rectified that.
+* There were some PHP notices popping up when a site has no posts we got rid of them.
+* The new Jetpack Ads feature will no longer be visible if your plan doesn't support them.
```
Comment (dereksmart) [inline readme.txt]: This isn't true -- the card is still there with the upgrade prompt.  We just made sure it gets deactivated if you cancel or downgrade.

File: readme.txt [hunk 3]
```diff
@@ -106,52 +106,53 @@ There are opportunities for developers at all levels to contribute. [Learn more
 = 4.6 =
 
 * Release date: February 8, 2017
```
Comment (dereksmart) [inline readme.txt]: Change this to feb 7

File: readme.txt [hunk 4]
```diff
@@ -106,52 +106,53 @@ There are opportunities for developers at all levels to contribute. [Learn more
 = 4.6 =
 
 * Release date: February 8, 2017
-* Release post:
+* Release post: http://wp.me/p1moTy-40g
 
 **Exciting New Features and Improvements**
 
-* New Google Analytics module. #5603
+* Enable Google Analytics and insert the necessary tracking code with this new Jetpack feature.
```
Comment (dereksmart) [inline readme.txt]: I like the notion of not having to touch a single line of code.  The module subheader reads `Set up Google Analytics without touching a line of code.` -- can we work that in here?

File: readme.txt (hunk 1 above)
Comment (jessefriedman) [inline readme.txt]: done

File: readme.txt (hunk 2 above)
Comment (jessefriedman) [inline readme.txt]: fixed

File: readme.txt (hunk 3 above)
Comment (jessefriedman) [inline readme.txt]: check

File: readme.txt (hunk 4 above)
Comment (jessefriedman) [inline readme.txt]: perfect, thanks

**Comment by dereksmart** [comment]
branch-4.6 af0621fdf9691d39b9babfe9d0870cb651212b85

**Pull request merged**